src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -667,14 +668,64 @@ def generate_pvt_model_compositional(value: DescriptionDocument):
     }
 
 
+def load_combined_fluid_description(
+    document: DescriptionDocument,
+) -> Dict[str, case_description.CombinedFluidDescription]:
+    alfacase_to_case_description = {
+        "pvt_model": load_value,
+    }
+
+    def generate_fluid_description(
+        value: DescriptionDocument,
+    ) -> case_description.CombinedFluidDescription:
+        case_values = to_case_values(value, alfacase_to_case_description)
+        item_description = case_description.CombinedFluidDescription(**case_values)
+        return update_multi_input_flags(document, item_description)
+
+    return {
+        key.data: generate_fluid_description(
+            DescriptionDocument(value, document.file_path)
+        )
+        for key, value in document.content.items()
+    }
```

Comment:
If you are going to make some more changes in this branch think of replacing this by

```suggestion
def load_combined_fluid_description(
    document: DescriptionDocument,
) -> Dict[str, case_description.CombinedFluidDescription]:
    return load_dict_of_instance(document,  case_description.CombinedFluidDescription)
```

I am sorry I didn't take my time to port all the loaders to use auto loaders so ppl get writing those abominations to get the test suite to pass.

Reply:
BTW, the same approach will work on `load_pvt_model_combined_description`. I will try to to add a change suggestion there too.

Reply:
Nice, very good suggestions. I love it!